oe_theme.theme

```diff
@@ -174,6 +174,14 @@ function oe_theme_theme_suggestions_field_alter(array &$suggestions, array $vari
     $suggestions[] = 'field__entity_reference_revisions__' . $element['#entity_type'] . '__' . $element['#bundle'] . '__' . $field_definition->getName();
   }
 
+  if (in_array($element['#field_type'], ['text_long', 'text_with_summary'])) {
```

Comment:
Why only for those two field types? The issue is about providing theme suggestion for field types in general, then to be used for those two specific types.